staging/src/k8s.io/kubectl/pkg/cmd/delete/delete.go

```diff
@@ -505,8 +505,10 @@ func (o *DeleteOptions) PrintObj(info *resource.Info) {
 
 	switch o.DryRunStrategy {
 	case cmdutil.DryRunClient:
+		kindString = fmt.Sprintf("%s %s", info.Namespace, kindString)
```

Comment:
nit:

Appending `namespace` into variable `kindString` which represents `group & kind` looks strange here. maybe a separated prefix logic below could be more clear.